test/test_nn_ops.cpp

```diff
@@ -118,7 +148,159 @@ TEST(NNOps, Conv2DBackpropFilter_NHWC) {
   }
 }
 
-}  // namespace testing
+// NCHW data format not supported on the test framework now
+TEST(NNOps, DISABLED_Conv2DBackpropFilterNCHW) {
```

Comment:
Can we move this disabled test to tf_Exec.cc temporarily? Let's not add any disabled test in the new framework, to avoid confusion. @avijit-nervana What do you think? We can add it back when the unit test framework supports it.

Reply:
That's a good suggestion. But this DISABLED test uses the new framework - so we probably should keep this here. We should retire the tf_exec.cpp once all the tests in there are converted to the new framework.